packages/components/stories/List.js

```diff
@@ -189,8 +189,7 @@ const props = {
 			},
 			{
 				id: 1,
-				name:
-					'Title with a lot of actions with super super super super super super super super super super super super super super super super super super super super super super super super super super super super super super long title oh yeah',
+				name: 'Title with a lot of actions',
```

Comment:
Renamed this one because we have an e2e test that search for it and I don't want to write `Title with super super super super super super super super super super super super super super super super super super super super super super super super super super super super super super long title oh yeah` instead